proxy/src/auth/backend/console_redirect.rs

```diff
@@ -29,8 +34,9 @@ pub(crate) enum ConsoleRedirectError {
 }
 
 #[derive(Debug)]
-pub struct ConsoleRedirectBackend {
+pub struct ConsoleRedirectBackend<'a> {
     console_uri: reqwest::Url,
+    api: MaybeOwned<'a, cplane_proxy_v1::AuthControlPlaneClient>,
 }
```

Comment:
As far as I can tell, we don't need the MaybeOwned trick here
```suggestion
pub struct ConsoleRedirectBackend {
    console_uri: reqwest::Url,
    api: cplane_proxy_v1::AuthControlPlaneClient,
}
```